### DISCUSSION TITLE
Flutter Development in GitHub Codespaces

### DISCUSSION THREAD

**Discussion by hashfx** [community/community#31029] [Codespaces]
#### Thanks in Advance

**Comment by jfullstackdev** [comment]
maybe this one 

https://verygood.ventures/blog/github-codespaces-and-run-flutter#:~:text=In%20order%20to%20run%20a,which%20includes%20the%20Flutter%20SDK.

**Reply by hashfx** [reply]
Thanks a lot for your assistance but I've already tried this one and it didn't worked🥲

Even after adding Flutter and Dart SDK to Path in Dockerfile, it still says "Flutter not found"

**Comment by treeder** [comment]
Follow the flutter installation instructions.
